deployments/kustomize/base/wordpress/deployment.yaml

```diff
@@ -89,6 +89,14 @@ spec:
         - name: nginx-configmap-volume
           mountPath: /etc/nginx/conf.d/nginx.conf
           subPath: nginx.conf
+        - name: plugins-themes
+          mountPath: /var/www/html/wp-content/plugins
+          subPath: wp-content/plugins
+          readOnly: true
+        - name: plugins-themes
```

Comment:
These have the same name, not sure if that's intentional.

Reply:
It's using the same volume (plugins-themes) twice, once for plugins and once for themes.

Reply:
It references the volume, just different mount paths.